pkg/apis/ceph.rook.io/v1/scc.go

```diff
@@ -26,15 +26,14 @@ import (
 
 // NewSecurityContextConstraints returns a new SecurityContextConstraints for Rook-Ceph to run on
 // OpenShift.
-func NewSecurityContextConstraints(name, namespace string) *secv1.SecurityContextConstraints {
+func NewSecurityContextConstraints(name string, namespaces ...string) *secv1.SecurityContextConstraints {
```

Comment:
```suggestion
func NewSecurityContextConstraints(name string, namespaces []string) *secv1.SecurityContextConstraints {
```
why not use a slice?

Reply:
It will be kind of a breaking change. In the end, we are achieving the same thing. and using variadic func gives more flexibility